lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -253,10 +253,11 @@ class MailboxDashBoardController extends ReloadableController {
       (failure) {
         log('MailboxDashBoardController::onData():failure $failure');
       },
-      (success) {
+      (success) async {
         log('MailboxDashBoardController::onData():success $success');
         if (success is SendingEmailState) {
           if (currentOverlayContext != null && currentContext != null) {
+            await Future.delayed(const Duration(seconds: 1));
```

Comment:
Should add option `delayTime` in `showToast` function to reuse for more case. Ex:
```
_appToast.showToastWithIcon(
    currentOverlayContext!,
    message: AppLocalizations.of(currentContext!).your_email_being_sent,
   delayTime: const Duration(seconds: 1)
);
```

Reply:
Greattt, thanks u